chainer/dataset/convert.py

```diff
@@ -9,6 +10,40 @@
 from chainer import utils
 
 
+class Converter(object):
```

Comment:
How about documenting this class and the `convert` method?

Reply:
Wrote docs.